pygame/rwobject.py

```diff
@@ -17,7 +69,18 @@ def rwops_encode_file_path(filepath):
 
 
 def rwops_from_file(fileobj):
-    rwops = sdl.SDL_RWFromFP(fileobj, 0)
+    try:
+        # We try use the SDL helper first, since
+        # it's the simplest code path
+        rwops = sdl.SDL_RWFromFP(fileobj, 0)
+    except (TypeError, IOError):
```

Comment:
Wonder a bit about how the two cases are checked here. If this matches what pygame does then bleh, but okay. Otherwise, I'm wondering whether this check isn't the cause of the two tests that we used to expect failures from not passing? Do we really need to catch both TypeError and IOError? IOError seems to imply that there is enough of the right type of object to attempt IO.